devel/build

```diff
@@ -3,12 +3,22 @@
 # Builds the nextstrain/base and nextstrain/base-builder images with useful
 # caching.
 #
+# By default this tags images using "latest", but you can provide a custom tag
+# name.
+#
 # Set CACHE_DATE in your environment to force layers after our custom cache
 # point to be re-built. See the ARG CACHE_DATE line in the Dockerfile for more
 # information.
 #
 set -euo pipefail
 
+# If the tag is not provided, default to "latest".
+if [[ $# -eq 0 ]]; then
+    tag="latest"
+else
+    tag=$1
+fi
```

Comment:
_non-blocking_

A more concise way of putting this is

```bash
tag="${1:-latest}"
```

Reply:
Done